src/sass/foundation/variables.scss

```diff
@@ -122,6 +122,43 @@ $buttonTerBorderColor:     $green3 !default;
 $buttonTerTextColor:       $buttonColor !default;
 $buttonTerDisabledTextColor:   darken($green3, 20%) !default;
 
+//Alternatives buttons
+$altButtonDefaultBgColor : $transparent !default;
+$altButtonDefaultBorder: $grayLightLight !default;
```

Comment:
The value of `$altButtonDefaultBorder ` should be `$buttonDefaultBorderColor` as its the same